packages/engine-core/src/tracing/runInChildSpan.ts

```diff
@@ -27,12 +27,20 @@ export async function runInChildSpan<R>(options: SpanOptions, cb: (span?: Span,
   if (options.active === false) {
     const span = tracer.startSpan(`prisma:client:${options.name}`, options, context)
 
-    return cb(span, context).finally(() => span.end())
```

Comment:
😿

Reply:
Had to be done because this function now accepts non-promise callbacks.